lib/commands.js

```diff
@@ -30,6 +30,7 @@ program.command('build [path]')
   .option('-e, --env [setting]', 'specify a set of override settings to apply')
   .option('-p, --production', 'same as `--env production`')
   .option('-d, --debug [pattern]', 'print verbose debug output to stdout')
+  .option('-J, --parallel [num]', 'parallelize the build')
```

Comment:
> It is better than storing that in the config, because number of workers that yield the speed increase depends on each machine, it's number of cores, so it doesn't belong to the config.

So why do you think it belongs in config? It's **machine dependent**, not so much **project dependent**, after all. It should be in the config. Some machines would be better off with no worker, some with 2, some with 4...

*(Make does it this way, [too](https://www.gnu.org/software/make/manual/html_node/Parallel.html))*